core/src/main/java/com/homeaway/streamplatform/streamregistry/health/StreamRegistryHealthCheck.java

```diff
@@ -178,10 +179,16 @@ private Stream createCanaryStream() {
     }
 
     private Schema createSampleSchema() {
+        org.apache.avro.Schema sampleSchema = SchemaBuilder.builder("com.streamplatform.healthcheck")
```

Comment:
We recently added Schema Validation. So, a valid schema is now necessary for inserting a Stream